reactjs/src/scenes/Roles/index.tsx

```diff
@@ -24,21 +24,29 @@ const confirm = Modal.confirm;
 @inject(Stores.RoleStore)
 @observer
 class Role extends AppComponentBase<IRoleProps> {
```

Comment:
Can you modify IRoleState as: 
export interface IRoleState {
  modalVisible: boolean;	  
  maxResultCount: number;	  
  skipCount: number;	  
  roleId: number;
  filter: string;
}	

and modify component definition as 
class Role extends AppComponentBase<IRoleProps, IRoleState>